src/Provider/Salesforce.php

```diff
@@ -102,4 +105,9 @@ protected function createResourceOwner(array $response, AccessToken $token)
     {
         return new SalesforceResourceOwner($response);
     }
+
```

Comment:
Please include a complete docblock description for this method. Since it appears to overload a method within the base package, you can copy and paste here.